code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -640,19 +650,37 @@
 	overdose_threshold = 35
 
 /datum/reagent/medicine/atropine/on_mob_life(mob/living/M)
-	if(M.health < 0)
+	if(M.health < -60)
+		M.adjustToxLoss(-4*REM)
+		M.adjustBruteLoss(-4*REM)
+		M.adjustFireLoss(-4*REM)
+		M.adjustOxyLoss(-10*REM)
+	else if(M.health < 0)
 		M.adjustToxLoss(-2*REM)
 		M.adjustBruteLoss(-2*REM)
 		M.adjustFireLoss(-2*REM)
 		M.adjustOxyLoss(-5*REM)
+	else if(M.health < 10)
+		M.adjustToxLoss(-0.5*REM)
+		M.adjustBruteLoss(-0.5*REM)
+		M.adjustFireLoss(-0.5*REM)
+		M.adjustOxyLoss(-0.5*REM)
+	if(istype(M, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = M
+		if(H.heart_attack && prob(10))
```

Comment:
prob 10 seems a bit too small for a medicine used exactly for heart attacks,i'd say atleast 25